### DISCUSSION TITLE
Does FreeMoCap run in real-time?

### DISCUSSION THREAD

**Discussion by abitrolly** [freemocap/freemocap#152] [Q&A] [answered]
Came here from https://github.com/freemocap/freemocap/wiki/FAQ which contains the same question, but is missing the answer.

**Comment by trentwirth** [comment, marked as answer]
not at the moment, it's something that devs are working on but it's still a ways off.

**Reply by abitrolly** [reply]
Can this help https://github.com/mit-han-lab/temporal-shift-module ?
